client/login/wp-login/private-site.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+
+class PrivateSite extends Component {
+	static propTypes = {
+		translate: PropTypes.func.isRequired,
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Card>
+				<div className="wp-login__private-image">
+					<img src="/calypso/images/private/private.svg" />
+				</div>
+
+				<h2 className="wp-login__private-header">
+					{ translate( 'This is a private WordPress.com site.' ) }
+				</h2>
+
+				<p>
+					{ translate( "Request an invitation to view it and we'll " +
+						'send your username to the site owner for their approval.' ) }
+				</p>
+
+				<Button primary className="wp-login__private-button">
+					{ translate( 'Request Invitation' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Invitation resent.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).